### ISSUE TITLE
how to use it in the arm-linux 

### ISSUE THREAD

**Issue by KingDamLiu** [FreeOpcUa/python-opcua#1067]
In my question,I can't cross compilling the lxml

**Comment by zerox1212** [comment]
I don't think this is a question for here. You need to get lxml to work with your platform. Read the docs for lxml, it has other dependencies.

**Comment by swamper123** [comment]
[[or use this quick link to their page](https://lxml.de/installation.html)]
libxml2 and libxslt has to be installed first.

**Comment by KingDamLiu** [comment]
of course! I have installed libxml2 and libxslt.but I don't kown how to setting the parameter to the command "python3 setup.py build" inlcude arm-linux-gcc and the any path.

**Comment by brubbel** [comment]
Can't you install via pip? It should build lxml for you if your specific arm architecture has no precompiled binaries available.
